code/modules/antagonists/traitor/objectives/destroy_machinery.dm

```diff
@@ -2,7 +2,7 @@
 	name = "Destroy Protolathe"
 	objectives = list(
 		/datum/traitor_objective/destroy_machinery = 1,
-		/datum/traitor_objective/destroy_machinery/high_risk = 1
+		/datum/traitor_objective/destroy_machinery/high_risk = 0
```

Comment:
Why?

Reply:
If you are replying to my comment: Because it can be done in a cleaner, modular manner.
If you are asking why remove this: Because protolathe destruction sounds unfun for all involved.